### PR TITLE
Run pod install

### PR DESCRIPTION
**Pull request by ladanazita**
This succeeds locally
`xcodebuild -scheme Segment-Amplitude_Example -workspace Example/Segment-Amplitude.xcworkspace`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,11 +18,5 @@ build:
 test:
 	xcodebuild test $(XC_ARGS) | $(XCPRETTY)
 
-xcbuild:
-	xctool $(XC_ARGS)
```
Comment (ladanazita) [inline Makefile:22]: Removed because xctool doesn’t support building for Xcode 8 and later. Facebook recommends updating to xcodebuild or other build tools. 
https://github.com/facebook/xctool/issues/704

**Review by f2prateek — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ DESTINATION ?= "platform=iOS Simulator,name=iPhone 5"
 PROJECT := Segment-Amplitude
 XC_ARGS := -scheme $(PROJECT)_Example -workspace Example/$(PROJECT).xcworkspace -sdk $(SDK) -destination $(DESTINATION) ONLY_ACTIVE_ARCH=NO
 
-install: Example/Podfile $(PROJECT).podspec
```
Comment (tonyxiao) [inline Makefile:8]: Why is this change needed?

**Review by tonyxiao — COMMENTED** [review]
LGTM, quick comments though.

File: Makefile (hunk 2 above)
Comment (ladanazita) [reply Makefile:8]: It wasn't - just consistent with other Makefiles in other integrations, I can change back - we have to redo Travis or move onto circle - so I will have a project to update Makefile scripts and travis/or circle builds and can address there
